python/test/unit/mesh/test_mesh_value_collection.py

```diff
@@ -166,3 +166,40 @@ def test_mesh_function_assign_2D_vertices():
         for i in range(num_cell_vertices):
             assert 25 == g.get_value(c, i)
             assert f2.values[vertices[i]] == g.get_value(c, i)
+
+
+def test_mvc_construction_array():
```

Comment:
Would be good to test quad/hex mesh and also MVC on vertices, can you add tests for this?

Reply:
Working on it

Reply:
done